tests/test_utils.py

```diff
@@ -2571,75 +2571,133 @@ def test_to_segmented_index(obj, allow_nat, files_duration, root, expected):
 
 
 @pytest.mark.parametrize(
-    'output_folder,table_id,expected_file_names',
+    'obj, expected_file_names',
     [
-        pytest.param(
-            '.',
-            'segments',
-            None,
-            marks=pytest.mark.xfail(raises=ValueError)
+        # empty
+        (
+            audformat.filewise_index(),
+            [],
         ),
-        pytest.param(
-            os.path.abspath(''),
-            'segments',
-            None,
-            marks=pytest.mark.xfail(raises=ValueError)
+        (
+            audformat.segmented_index(),
+            [],
+        ),
+        (
+            pd.Series(index=audformat.filewise_index(), dtype='object'),
+            [],
+        ),
+        (
+            pd.Series(index=audformat.segmented_index(), dtype='object'),
+            [],
         ),
+        # frame
         (
-            'tmp',
-            'segments',
+            pytest.DB['segments'].get(),
             [
-                str(i).zfill(3) + f'_{j}'
-                for i in range(1, 11)
-                for j in range(10)
-            ]
+                str(i + 1).zfill(3) + f'_{j}'
```

Comment:
Could you maybe add a comment above this statement what this generates as I have a hard time understanding it, e.g.

```python
# f1_0, f1_1, ...
```

Reply:
I had copied it from the previous test. I now switched to `f'...'` notation and also added a comment.

Reply:
Thanks.